src/test/java/uk/gov/ons/ssdc/caseprocessor/messaging/RefusalReceiverIT.java

```diff
@@ -64,7 +66,7 @@ public void setUp() {
 
   @Test
   public void testRefusal() throws Exception {
-    try (QueueSpy rhCaseQueueSpy = rabbitQueueHelper.listen(RH_CASE_QUEUE)) {
+    try (QueueSpy outboundCaseQueueSpy = rabbitQueueHelper.listen(RH_CASE_QUEUE)) {
```

Comment:
still using the original constant in all-caps as if it was `static final`... I think it _should_ be using a constant anyway, so this line is good to me.